Yosemite/Yosemite/Tools/POS/POSOrderService.swift

```diff
@@ -58,6 +63,17 @@ public final class POSOrderService: POSOrderServiceProtocol {
         }
         return syncedOrder
     }
+
+    public func sendOrderReceipt(order: Order, toEmailAddress: String) async throws {
+        guard order.billingAddress?.email == nil || order.billingAddress?.email == "" else {
+            throw NSError(domain: "Email already set", code: 0)
+        }
+        let updatedBillingAddress = order.billingAddress?.copy(email: toEmailAddress)
```

Comment:
The name of the `toEmailAddress` parameter makes this line difficult to read. Reading it like a sentence, it appears to say "copy the billing address to the email address", when actually it's more like the opposite of that.

Reply:
Changed to `recipientEmail`: https://github.com/woocommerce/woocommerce-ios/pull/14481/commits/781b3ce8cdc6a36ac33b93ef6ef47516acdf5f37